functions/Get-DbaServerRoleMember.ps1

```diff
@@ -99,7 +99,7 @@ function Get-DbaServerRoleMember {
             Write-Message -Level Verbose -Message "Attempting to connect to $instance"
 
             try {
-                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential
+                $server = Connect-DbaInstance -SqlInstance $instance -SqlCredential $SqlCredential
```

Comment:
Unless we are going to do this across all the commands, this should stay with internal function.

Reply:
I will merge then set it back to be included in the mass change if we make it. Thanks for the review @jpomfret and @wsmelton !